src/app/feature/consultant/components/consultant-header/consultant-header.component.scss

```diff
@@ -103,6 +116,11 @@ $edit-button-position-sm: 6px;
 }
 
 .edit-button {
```

Comment:
Without running the code and trying this I believe you can just combine these two classes:

`
.edit-button, .export-button {
    position: absolute;
    z-index: 999;
}
`

Reply:
Can't combine these, one is z-index: 99**8**, the other is 99**9**.  I tried having them the same but it overlaid the buttons for some reason and messed up the layout.